Periodic_4_hyperbolic_triangulation_2/test/Periodic_4_hyperbolic_triangulation_2/test_p4ht2_matrix.cpp

```diff
@@ -0,0 +1,47 @@
+#include <CGAL/CORE_Expr.h>
+#include <CGAL/Point_2.h>
+#include <CGAL/Circle_2.h>
+#include <CGAL/Cartesian.h>
+#include <CGAL/internal/Exact_complex.h>
+#include <CGAL/internal/Hyperbolic_octagon_translation_matrix.h>
+
+#include <iostream>
+#include <vector>
+
+using namespace CGAL;
+using namespace std;
```

Comment:
Do not use `using`

Reply:
In a .cpp test file, why not?

Reply:
You certainly know better.  Would you also put a `using`  in CGAL examples?  I always had the feeling that it invites trouble for the disambiguation.

Reply:
No, I would forbid the same code in an example, because they are part of the CGAL manual, and those using directives (on full namespaces) are generally a bad practice.

Reply:
I have removed the `using` statements.